src/styles/components/_services.scss

```diff
@@ -0,0 +1,91 @@
+.our-services {
+  background-color: #f2f2f2;
+  border-bottom: 6px solid #ffffff;
+  padding: 59px 0px 26px 0px;
+
+  @include media-tablet {
+    padding: 39.62px 0px 25px 0px;
+    border-bottom: none;
+  }
+
+  @include media-mobile {
+    padding: 25.6px 0px 0px 0px;
+    border-bottom: none;
+  }
+}
+
+.our-services-wrapper {
+  max-width: $content-width;
+  margin: auto;
+
+  p {
+    color: #767e9e;
+  }
+}
+
+.layout-3-colum {
+  margin: 0px 40px;
+  @include flex(31.91%);
+
+  @include media-tablet {
+    margin: 0px 75px 0px 42px;
+    @include flex(46.13%);
+  }
+
+  @include media-mobile {
+    margin: 0px 43px 0px 33px;
+    @include flex(100%);
+  }
+}
+
+.services-item {
+  width: 300px;
+  display: flex;
+
+  .service-item-description {
```

Comment:
https://stackoverflow.com/questions/36652580/how-to-add-three-dots-to-text-when-overflow-in-html